org.kde.kleopatra.json

```diff
@@ -0,0 +1,91 @@
+{
+  "app-id": "org.kde.kleopatra",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "kleopatra",
+  "rename-icon": "kleopatra",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--filesystem=~/.gnupg:create"
```

Comment:
I'm not sure this is safe. What happens when the keyring gets accessed from outside and inside a Flatpak at the same time?

Reply:
Interesting. Maybe this will work.